app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -238,6 +243,19 @@ class BrowserFragment :
             tabId = tab.id
         ), this, view)
 
+        appLinksFeature.set(
+            feature = AppLinksFeature(
+                requireContext(),
+                store = components.store,
+                sessionId = tabId,
+                fragmentManager = parentFragmentManager,
+                launchInApp = { true },
```

Comment:
This will always show a dialog before switching to a third-party app, right?

Reply:
Yes, I verified that locally.